src/components/OfferList/OfferList.scss

```diff
@@ -0,0 +1,13 @@
+.offer-list {
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 20px;
+
+  .offer-row {
+    flex-direction: column;
+    width: 1000px;
```

Comment:
Since it's bootstrap's `Row` - it should be already on full width of `Container`, isn't it? Then it would display correctly on various resolutions.